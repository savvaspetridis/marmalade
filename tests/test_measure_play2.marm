/* TEST MEASURE PLAY w/ default time */

measure a_1 = $() [44.e, 45.e, 46.e, 47.e, 48.e, 49.e, 50.e, 51.e];

(play()) [a_1];
