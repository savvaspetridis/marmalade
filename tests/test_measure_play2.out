jMusic Play: Playing score Untitled Phrase score using JavaSound General MIDI soundbank.
jMusic Play: Waiting for the end of Untitled Phrase score.
jMusic MidiSynth: Stopped JavaSound MIDI playback